libraries/common/io/files.effekt

```diff
@@ -89,17 +89,17 @@ def filesystem[R] { program: => R / Files }: R / { AsyncIO, Exception[IOError] }
 /**
  * Reads a file at given path as utf8 encoded string.
  */
-def readFile(path: String): String / { AsyncIO, Exception[IOError] } = {
-  val fd = openFile(path, ReadOnly());
-  with on[IOError].finalize { closeFile(fd) }
+def readFile(path: String): String / Exception[IOError] = {
+  val fd = direct::openFile(path, modeName(ReadOnly()));
+  with on[IOError].finalize { direct::closeFile(fd); () }
```

Comment:
It's nice that we can make this work in LLVM, but in JS this sadly cannot work. There, you cannot make a synchronize file operation asynchronous.

Reply:
Ok, I retract. Of course we can, but this will poison the coeffecrs with `control` (which is maybe fine)